src/search.cpp

```diff
@@ -258,6 +278,17 @@ void GreedySearch_Cpu::AppendNextTokensToSequences() {
   }
 }
 
+bool BeamSearch_Cpu::IsDone() const {
+  if (beam_scorer_->IsDone()) {
+    return true;
+  } else if (sequences_.GetSequenceLength() == params_->search.max_length) {
+    if (g_log.enabled && g_log.hit_max_length)
+      Log("hit_max_length", "beam cuda hit");
```

Comment:
what is "beam cuda hit"?

Reply:
The first parameter needs to be label and 2nd parameter is the content: https://github.com/microsoft/onnxruntime-genai/blob/616690213be076665f965842b01955c021f32334/src/logging.cpp#L80